### PR TITLE
sql-diagnostic response

### PR DESCRIPTION
**Pull request by treep78**
add responses in diagnostic.sql

Comfort: 3
Clarity: 4

I used Google and my class notes to help with this diagnostic.

### CODE REVIEW

File: diagnostic.sql [hunk 1]
```diff
@@ -5,24 +5,47 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname, given_name, gender, height, weight, born_on, eye_color, hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
+DROP TABLE IF EXISTS licensees;
+
```
Comment (BenGitsCode) [inline diagnostic.sql:9]: 👍

File: diagnostic.sql [hunk 2]
```diff
@@ -5,24 +5,47 @@ CREATE DATABASE "sql-crud-assessment";
...
+SET height = height-1/12
+WHERE surname LIKE '%Escobar'
+AND WHERE surname LIKE '%Whaley';
 
 -- Question 5. Remove Dylan Rich and Teresita Myers from the table of licensees.
 
--- your code here
+DELETE FROM licensees
+WHERE surname LIKE '%Rich'
+AND WHERE surname LIKE '%Myers';
```
Comment (BenGitsCode) [inline diagnostic.sql:51]: These are right except for the `%`'s Why did you add those?

**Comment by treep78** [comment]
I saw them in an example.

**Comment by BenGitsCode** [comment]
Ah okay. Well as far as I know, that isn't valid SQL and not the conventional syntax we'll be using for this course. If you want to provide the example, I'm happy to look at it.

**Review by BenGitsCode — COMMENTED** [review]
3/5

good work!
